src/pages/using-spark/components/autocomplete.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Autocomplete
+---
+
+import ComponentPreview from '../../../components/ComponentPreview';
+import { SprkDivider } from '@sparkdesignsystem/spark-react';
+import typingExample from '../../../images/autocomplete/Typing.png';
+import selectingExample from '../../../images/autocomplete/SelectingOption.png';
+import noResultsFoundExample from '../../../images/autocomplete/NoMatches.png';
+import noSearchIconExample from '../../../images/autocomplete/NoSearchIcon.png';
+import twoLinesExample from '../../../images/autocomplete/Addresstwolines.png';
+import noLabelExample from '../../../images/autocomplete/NoLabelAbove.png';
+import categoriesExample from '../../../images/autocomplete/Searchacrosscategories.png';
+import recentSearchesExample from '../../../images/autocomplete/Focused_withoutsearch.png';
+
+# Autocomplete
+
+Autocomplete input fields allow the browser to predict information being
+entered by the user. The browser should display options based on
+previous information entered.
+
+<ComponentPreview
+  componentName="autocomplete--default-story"
+  hasHTML
+  minHeight="27rem"
+/>
+
+### Usage
+
+Autocomplete helps users search for options that are difficult to remember or
+commonly misspelled. They allow designers to include more options to choose
+from, with less space.
+
+### Guidelines
+
+Autocomplete input fields should be used:
+
+- When information can be predicted from previously entered information.
+- When accuracy and speed are important to the user.
+- When the number of options is no longer convenient or efficient to display
+  in a select box.
+- When a select box has too many options but can be predicted for the user
```

Comment:
We'll wanna double check some of these bullet points for content. Some have periods on the end and some don't.